1-js/02-first-steps/08-comparison/article.md

```diff
@@ -1,95 +1,95 @@
-# Comparisons
+# Comparações
 
-We know many comparison operators from maths:
+Conhecemos muitos operadores de comparação de matemática:
 
-- Greater/less than: <code>a &gt; b</code>, <code>a &lt; b</code>.
-- Greater/less than or equals: <code>a &gt;= b</code>, <code>a &lt;= b</code>.
-- Equals: `a == b` (please note the double equals sign `=`. A single symbol `a = b` would mean an assignment).
-- Not equals. In maths the notation is <code>&ne;</code>, but in JavaScript it's written as an assignment with an exclamation sign before it: <code>a != b</code>.
+- Maior/menor que: <code>a &gt; b</code>, <code>a &lt; b</code>.
+- Maior/menor ou igual que: <code>a &gt;= b</code>, <code>a &lt;= b</code>.
+- Igual: `a == b` (observe o sinal de igual duplo `=`. Um único sinal de igual `a = b` significaria uma atribuição).
+- Diferente. Em matemática, a notação é <code>&ne;</code>, mas em JavaScript é escrita como uma atribuição com um sinal de exclamação antes: <code>a != b</code>.
 
-## Boolean is the result
+## Booleano é o resultado
 
-Like all other operators, a comparison returns a value. In this case, the value is a boolean.
+Como todos os outros operadores, uma comparação retorna um valor. Nesse caso, o valor é um booleano.
 
-- `true` -- means "yes", "correct" or "the truth".
-- `false` -- means "no", "wrong" or "not the truth".
+- `true` -- significa "sim", "correto" ou "verdade".
+- `false` -- significa "não", "errado" ou "falso".
 
-For example:
+Por exemplo:
 
 ```js run
 alert( 2 > 1 );  // true (correct)
 alert( 2 == 1 ); // false (wrong)
 alert( 2 != 1 ); // true (correct)
 ```
 
-A comparison result can be assigned to a variable, just like any value:
+Um resultado de comparação pode ser atribuído a uma variável, assim como qualquer valor:
 
 ```js run
 let result = 5 > 4; // assign the result of the comparison
 alert( result ); // true
 ```
 
-## String comparison
+## Comparação de strings
 
-To see whether a string is greater than another, JavaScript uses the so-called "dictionary" or "lexicographical" order.
+Para ver se uma string é maior que outra, o JavaScript usa o chamado ordem "dicionário" ou "lexicográfico".
```

Comment:
'a chamad**a** ordem "dicionário" ou lexicográfi**a**".'
"ordem" is a feminine noun in portuguese.